LSpec/LSpec.lean

```diff
@@ -18,12 +19,8 @@ inductive TestSeq
   | more : String → (prop : Prop) → TDecidable prop → TestSeq → TestSeq
   | done
 
-/-- `test` is a single basic test. -/
-def test (descr : String) (p : Prop) [TDecidable p] : TestSeq :=
-  .more descr p inferInstance .done
```

Comment:
Did we never actually need to separate out the `test` and `test'` functions?

Reply:
No! hehehe